Resolve issue #13

Enoch-Kim opened the pull request:
* index.html All Stories -> Posts로 이름 변경
* default.html : title 삭제하고 full_logo로 헤더 변경
* dsc_full_logo 로 asset name change
* full_logo class 로 로고 중앙 디스플레이
** 제꺼 머지하실때 rebase merge 부탁드립니다.
Resolve #13

Enoch-Kim committed:
* _config.yml : full_logo 선언
* index.html All Stories -> Posts로 이름 변경
* default.html : title 삭제하고 full_logo로 헤더 변경
* dsc_full_logo 로 asset name change
* full_logo class 로 로고 중앙 디스플레이

Enoch-Kim committed:
* _config.yml : full_logo 선언
* index.html All Stories -> Posts로 이름 변경
* default.html : title 삭제하고 full_logo로 헤더 변경
* dsc_full_logo 로 asset name change
* full_logo class 로 로고 중앙 디스플레이

MOBUMIN commented on assets/css/screen.css:
```diff
@@ -3,6 +3,12 @@ Template Name: Mediumish
+.full_logo {
+    margin: auto;
+    display: block;
+    margin-bottom: 50px;
```
margin-bottom 있는게 이쁜가여?

Enoch-Kim replied:
넵 좀 공간을 두는게 이쁘더라구요

Enoch-Kim replied:
리베이스 머지는 https://meetup.toast.com/posts/122 참고해주세요
그냥 전 브랜치 없애는게 좋아서 그렇습니다...ㅎㅎ

Enoch-Kim replied:
사실 Squash and Merge가 맞는건데, commit 한개라 rebase랑 동일해서요

Enoch-Kim replied:
머지하는 곳 옆에 보시면 머지 방식 지정할 수 있어요
<img width="980" alt="스크린샷 2020-09-27 오후 9 14 58" src="https://user-images.githubusercontent.com/60127664/94364734-83f16900-0106-11eb-80db-3304aeb71144.png">

MOBUMIN replied:
아하 설명이랑 링크 감사합니다! ㅎㅎ

MOBUMIN reviewed (dismissed):
굿굿
margin-bottom 50px나 있는 이유가 궁금합니다! 있는게 낫다면 이대로 머지하면 될듯싶어요 :D

++ rebase merge 할 줄 몰라서 호다닥 approve 합니다

jeongukjae commented on _layouts/default.html:
```diff
@@ -2,7 +2,6 @@
 <html>
 
 <head>
-    <title>{{ page.title }} | {{ site.name }}</title>
```
어라 타이틀은 왜 지운거예요?? 다른 타이틀 태그를 적용할 계획인가요??

Enoch-Kim replied:
엥?! 이건 제 실수 입니다. 꼼꼼한 리뷰 감사해요!

jeongukjae commented on _layouts/default.html:
```diff
@@ -94,10 +93,11 @@
+                <!-- <p class="lead">
                     {{ site.description }}
-                </p>
+                </p> -->
```
주석 처리는 필요없어요 :) 지워주셔도 됩니다.

Enoch-Kim committed:
* _config.yml 파일의 feed plugin 제거

Enoch-Kim committed:
* _config.yml 파일의 feed plugin 제거

MOBUMIN commented on _config.yml:
```diff
@@ -7,14 +7,14 @@ include: ["_pages"]
 plugins:
   - jekyll-paginate
   - jekyll-sitemap
-  - jekyll-feed
```
오잉 feed는 왜 없어졌나유??

Enoch-Kim replied:
@MOBUMIN 저희 feed 연결 안하고 삭제하기로 했었습니다. (feed.xml 도 이미 삭제했어요 ㅎㅎ)

MOBUMIN replied:
헐ㄹ랭

jeongukjae replied:
엥 어떤 의사결정이 들어갔었나요?? feed.xml 삭제하는 것은 괜찮은데 jekyll feed 삭제하는 것은 이상한 것 같아요.

Enoch-Kim replied:
엥 저번에 feed 필요없을 것 같다는 말이 feed.xml만을 의미하셨던 건가요..
@MOBUMIN 죄송합니다. 제가 오해했네요 ㅠㅠ
그럼 돌려놓겠습니다.

Enoch-Kim committed:
Merge branch 'issue13' of github.com:DSC-University-of-Seoul/dsc-university-of-seoul.github.io into issue13

MOBUMIN reviewed (dismissed):
굿굿 굿굿 굿 굿

jeongukjae reviewed (changes requested):
나머지는 좋아보이지만 jekyll-feed 삭제와 title tag 삭제, 그리고 주석 처리는 이상한 것 같아요.

Enoch-Kim committed:
Accept Review
* draw back jekyll feed
* draw back page title
* remove useless comment

Enoch-Kim commented:
> 나머지는 좋아보이지만 jekyll-feed 삭제와 title tag 삭제, 그리고 주석 처리는 이상한 것 같아요.

수정 반영했습니다~

MOBUMIN reviewed (dismissed):
file chaneged에 이전에 바뀌었던 항목들도 +-로 잡히는데 충돌 때문에 그러신 건가요?-? ( 잘 몰라서 물어보는 거에요! )

Enoch-Kim commented:
> file chaneged에 이전에 바뀌었던 항목들도 +-로 잡히는데 충돌 때문에 그러신 건가요?-? ( 잘 몰라서 물어보는 거에요! )

네.. 그런것 같아요.. 충돌 해결해야겠네요..

MOBUMIN commented:
> > file chaneged에 이전에 바뀌었던 항목들도 +-로 잡히는데 충돌 때문에 그러신 건가요?-? ( 잘 몰라서 물어보는 거에요! )
> 
> 네.. 그런것 같아요.. 충돌 해결해야겠네요..

오노.... 해결해서 그렇게 뜨는 건줄 알았는데 해결이 안 돼서 그런 거였나요...!!!

Enoch-Kim commented:
> > > file chaneged에 이전에 바뀌었던 항목들도 +-로 잡히는데 충돌 때문에 그러신 건가요?-? ( 잘 몰라서 물어보는 거에요! )
> > 
> > 
> > 네.. 그런것 같아요.. 충돌 해결해야겠네요..
> 
> 오노.... 해결해서 그렇게 뜨는 건줄 알았는데 해결이 안 돼서 그런 거였나요...!!!

이게 image bot이 master의 로고를 최적화해서 발생했네요..
우선 master의 logo파일을 추가해줘서 충돌은 해결했습니다. 나중에 이 파일 삭제해서 master에 추가해야겠네요

Enoch-Kim commented:
실수로 눌렀습니다..ㅠㅠ

Enoch-Kim commented:
> > > file chaneged에 이전에 바뀌었던 항목들도 +-로 잡히는데 충돌 때문에 그러신 건가요?-? ( 잘 몰라서 물어보는 거에요! )
> > 
> > 
> > 네.. 그런것 같아요.. 충돌 해결해야겠네요..
> 
> 오노.... 해결해서 그렇게 뜨는 건줄 알았는데 해결이 안 돼서 그런 거였나요...!!!

Image bot 이 로고 최적화해서 충돌이 생겼었네요.
우선 master의 최적화된 로고를 추가해줬어요.
TODO : DSC University of Seoul Light Vertical-Logo.png 삭제

Enoch-Kim committed:
* resolve conflict
    - img bot 이 master에서 변경한 파일이 충돌 발생
    - master의 image를 임시로 추가
    - TODO: DSC University of Seoul Light Vertical-Logo.png 제거

MOBUMIN reviewed (dismissed):
LGTM

jeongukjae reviewed (dismissed):
:+1:

jeongukjae committed:
Merge branch 'master' into issue13
